template/src/stores/Store.js

```diff
@@ -0,0 +1,92 @@
+import Reflux from 'reflux';
```

Comment:
In the case of React it's an Eslint AirBnb code style guide requirement to use PascalCase for imports in the case of classes (http://airbnb.io/javascript/#naming--PascalCase). I assumed, and possibly incorrectly that the default import of Reflux was also a class, or at least an instantiated object.

TL;DR, I don't have a good reason for or against. Based on our code style guides what you prefer the syntax to resemble.

Reply:
Discussed amongst Sydney devs and majority would prefer to keep all file names [slug-case](https://github.com/ianstormtaylor/to-slug-case) (i.e. all lower-case and with dashes).

Reasons for the change:
- reduced mental mapping from slug-case to UpperCamelCase (the component name)
- webpack can warn of case-insensitive imports

Reasons against change:
- all other mongodb-js modules follow this convention
- potential [issues](https://github.com/airbnb/javascript/issues/501) with case-insensitive file systems

Reply:
- [ ] Update the filenames (use `git mv` to achieve this otherwise git won't recognise the change in name in the case of case insensitive filesystems like Mac OSX)